examples/webots/vacuum/vacuum_simple.scenic

```diff
@@ -0,0 +1,146 @@
+"""
+Generate a simple room for the i-roomba create vacuum
+"""
+
+model scenic.simulators.webots.model
+
+import numpy as np
+import trimesh
+import random
+from pathlib import Path
+
+param numToys = 0
+param duration = 10
+
+## Class Definitions ##
```

Comment:
Since these are common to both vacuum scenarios, you might consider extracting them into a separate file imported by both: then it would be easier to see the differences between the two scenarios and to write additional scenarios along the same lines.